tests/bin/identities.test.ts

```diff
@@ -135,7 +135,7 @@ describe('CLI Identities', () => {
     keynode.id = polykeyAgent.nodeManager.getNodeId();
 
     testProvider = new TestProvider();
-    await polykeyAgent.identitiesManager.registerProvider(testProvider);
+    polykeyAgent.identitiesManager.registerProvider(testProvider);
```

Comment:
This can indeed be done. But I was actually thinking that you do this before `PolykeyAgent.createPolykeyAgent` but dependency injecting the `identitiesManager`. 

But due to the complexity of setting up the `identitiesManager` prior, I guess this can do for now.

Reply:
Actually maybe we should do this for the global agent instead? We can't use any of the existing real providers anyway.